manifests/mod/ssl.pp

```diff
@@ -97,6 +100,7 @@
   Optional[String] $stapling_cache                          = undef,
   Optional[Boolean] $ssl_stapling_return_errors             = undef,
   $ssl_mutex                                                = undef,
+  Boolean $ssl_reload_on_change                             = false,
```

Comment:
Should this default to undef and then use `pick($ssl_reload_on_change, $apache::ssl_reload_on_change)`?

Reply:
No, that makes no sense, see also our further discussions